src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -445,11 +458,15 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
             IReadOnlyDictionary<string, CentralPackageVersion> centralPackageVersions = null;
             if (isCpvmEnabled)
             {
-                // Add the central version information and merge the information to the package reference dependencies
+                // Add the central versionString information and merge the information to the package reference dependencies
                 centralPackageVersions = GetCentralPackageVersions();
                 packageReferences = ApplyCentralVersionInformation(packageReferences, centralPackageVersions);
             }
 
+            IReadOnlyDictionary<string, PrunePackageReference> packagesToPrune = MSBuildStringUtility.IsTrue(_vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.RestoreEnablePackagePruning)) ?
```

Comment:
For some reason I can just make the suggestion here but I think this format is easier to read for the ternary operator with the `?` and `:` at the beginning of the line instead of the end:

```cs
IReadOnlyDictionary<string, PrunePackageReference> packagesToPrune = MSBuildStringUtility.IsTrue(_vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.RestoreEnablePackagePruning))
                ? GetPackagesToPrune()
                : ImmutableDictionary<string, PrunePackageReference>.Empty;
```

Just my opinion.